docker/build.gradle

```diff
@@ -96,7 +98,8 @@ task quickstartDebug(type: Exec, dependsOn: ':metadata-ingestion:install') {
             'datahub docker quickstart',
             '--no-pull-images',
             '--version', "debug",
-            '--dump-logs-on-failure'
+            '--dump-logs-on-failure',
+            '-f ../docker/quickstart/docker-compose-without-neo4j-m1.quickstart.yml'
```

Comment:
This will break gradlew quickstart for non m1 users.

Reply:
Ah didn't mean to commit this, will remove. Local setting

Reply:
Thanks for catching

Reply:
Fixed